packages/contracts-bedrock/book/src/policies/versioning.md

```diff
@@ -107,15 +107,18 @@ discouraged. If a change is required to a release candidate after it has been ta
 
 ### Additional Release Candidates
 
-Sometimes additional release candidate versions are needed, in that case, the follow process should be used.
-This process is designed to (1) ensures fixes are made on both the release and the trunk branch
- and (2) avoids the need to stop development efforts on the trunk branch.
+Sometimes fixes or additional changes need to be added to a release candidate version. In that case
+we want to ensure fixes are made on both the release and the trunk branch, without stopping development
+efforts on the trunk branch.
 
+The process is as follows:
 
-1. Make the fixes on `develop`. *For whatever the normal semver level increment should be, bump that value by 5.*
+1. Make the fixes on `develop`. Increment the contracts semver as normal.
 2. Create a new release branch, named `proposal/op-contracts/X.Y.Z-rc.n+1` off of the rc tag.
-3. Cherry pick the fixes from `develop` into that branch. *Bump the semvers as normal, ensuring that the resulting version is less than the one on `develop`.
-4. After merging the changes into the new release branch, tag the resulting commit on the proposal branch as `op-contracts/vX.Y.Z-rc.2`.
+3. Cherry pick the fixes from `develop` into that branch. Instead of incrementing the semver as normal,
+   append `-patch.n` to the end of the version number. The value of `n` should start at 1 and be
+   incremented for each additional patch.
+4. After merging the changes into the new release branch, tag the resulting commit on the proposal branch as `op-contracts/vX.Y.Z-rc.n+1`.
```

Comment:
Why did the tag name here change from `op-contracts/vX.Y.Z-rc.n` to `op-contracts/vX.Y.Z-rc.n+1`? I think we still want `op-contracts/vX.Y.Z-rc.n`?

Reply:
It goes up by 1 everytime you do this operation, but I'm fine to remove it. People know what `n` should be.

```suggestion
4. After merging the changes into the new release branch, tag the resulting commit on the proposal branch as `op-contracts/vX.Y.Z-rc.n`.
```

Reply:
Ohh I interpreted that literally as meaning e.g. `op-contracts/vX.Y.Z-rc.3+1` as the tag name lol

Reply:
oh haha